### ISSUE TITLE
Collapse step didn't output a gtf file without a reference annotation

### ISSUE THREAD

**Issue by dudududu12138** [BrooksLabUCSC/flair#339]
Hi, I recently ran flair in the unguided mode, that is, I didn't offer a reference annotation. There are no error message. But there are only 2 outputs: 1)`isoforms.bed`,2)`isoforms.fa`. The `isoforms.gtf` was missing. Do you know the reason? Below are my codes:
```
bam2Bed12 -i ../minimap2/$sample.sorted.bam >$out/$sample.bed

flair collapse -t 10 \
        -q $input/${sample}.bed \
        -r ../reads/$sample/$sample.fastq \
        -g $ref \
        -o $output/$sample/$sample
```
By the way, I also ran `flair collapse` with a reference annotation. I followed your pipeline. But there are many other outputs. Could you explain the outputs except of `stringent.isoforms.bed`,`stringent.isoforms.fa` and `stringent.isoforms.gtf`? Below are the outputs and my codes :
```
stringent.annotated_transcripts.alignment.counts
stringent.annotated_transcripts.bed
stringent.annotated_transcripts.fa
stringent.annotated_transcripts.isoform.read.map.txt
stringent.annotated_transcripts.supported.bed
stringent.combined.isoform.read.map.txt
stringent.isoform.read.map.txt
stringent.isoforms.bed
stringent.isoforms.fa
stringent.isoforms.gtf
```
```
flair collapse -g genome.fa --gtf gene_annotations.gtf -q reads.flair_all_corrected.bed -r reads.fastq
--stringent --check_splice --generate_map --annotation_reliant generate
```
Waiting for your reply. Thanks!

**Comment by cafelton** [comment]
Yes, FLAIR collapse only generates a gtf when given an input gtf. I will fix this in the next release and update you. 
I think stringent is your output prefix?

These are the final output files:
stringent.isoforms.bed
stringent.isoforms.fa
stringent.isoforms.gtf
stringent.combined.isoform.read.map.txt

These are files related to only detected annotated transcripts from the gtf
stringent.annotated_transcripts.alignment.counts
stringent.annotated_transcripts.bed
stringent.annotated_transcripts.fa
stringent.annotated_transcripts.isoform.read.map.txt
stringent.annotated_transcripts.supported.bed

This is only reads supporting novel transcripts
stringent.isoform.read.map.txt
